components/scream/src/physics/shoc/shoc_assumed_pdf_impl.hpp

```diff
@@ -152,13 +152,13 @@ void Functions<S,D>::shoc_assumed_pdf(
                  ekat::min(100,
                            ekat::max(0, (3*tmp_val_1*(1 - a*ekat::square(tmp_val_2) - (1-a)*ekat::square(tmp_val_1))
                                          - (Skew_thl - a*ekat::cube(tmp_val_2) - (1 - a)*ekat::cube(tmp_val_1)))
-                                         /(3*a*(tmp_val_1 - tmp_val_2))*thlsec)));
+                                         /(3*a*(tmp_val_1 - tmp_val_2))))*thlsec);
```

Comment:
This error was not caught in the shoc_assumed_pdf bfb test since the condition below was never triggered. I changed the input grid in the test so that the condition here is met.